deployments/scripts/setup-k3d.sh

```
@@ -19,6 +19,12 @@ if ! command -v kubectl &> /dev/null; then
     exit 1
 fi
 
++if ! command -v helm &> /dev/null; then
```

Shellcheck (0.11.0)

[error] Parsing stopped here. Is this keyword correctly matched up?

(SC1089)